packages/office-ui-fabric-react/src/utilities/positioning/positioning.ts

```diff
@@ -335,12 +334,31 @@ function _adjustFitWithinBounds(
       const flippedElementEstimate = _flipAlignmentEdge(elementEstimate, target, gap, coverTarget);
       if (_isRectangleWithinBounds(flippedElementEstimate.elementRectangle, bounding)) {
         return flippedElementEstimate;
+      } else {
+        // If the flipped elements edges are still out of bounds, try nudging it.
+        elementEstimate = _alignOutOfBoundsEdges(
```

Comment:
could you add test coverage here?

Reply:
@joschect I only see a positioning.test.ts file where i can go and add some unit tests to validate the scenario. Are there additional places to add tests for positioning. Any visual regression ones?

Reply:
@srishtis27 Unit tests are all positioning has and do a good job managing that. It does already get visual regression coverage in the callout and contextualmenu tests.